trace-dispatcher/src/Cardano/Logging/Types.hs

```diff
@@ -53,7 +53,7 @@ module Cardano.Logging.Types (
 
 
 import           Codec.Serialise (Serialise (..))
-import           Data.Aeson ((.=))
+-- import           Data.Aeson ((.=))
```

Comment:
Please remove commented out line (+ the ones below).

Reply:
Done